x/bank/src/errors.rs

```diff
@@ -1,25 +1,23 @@
+use std::num::NonZero;
+
 use gears::{
-    application::handlers::node::{ErrorCode, ModuleInfo, TxError},
-    error::AppError,
+    application::handlers::node::{ModuleInfo, TxError},
+    x::errors::BankKeeperError,
 };
 use thiserror::Error;
 
 #[derive(Error, Debug)]
 pub enum BankTxError {
     #[error(transparent)]
-    Other(#[from] AppError), //TODO: stop using AppError
+    Keeper(#[from] BankKeeperError),
 }
 
 impl BankTxError {
     pub fn into<MI: ModuleInfo>(self) -> TxError {
         let code = match &self {
-            BankTxError::Other(_) => 1,
+            BankTxError::Keeper(_) => 1,
```

Comment:
We could use the NonZero macro here to avoid unwrapping later on.

Reply:
Done